extensions/hl7/runtime/src/main/doc/usage.adoc

```diff
@@ -0,0 +1,3 @@
+For MLLP with TCP, Netty is the only supported means of running an Hl7 MLLP listener. Mina is not supported since it has no GraalVM native support at present.
```

Comment:
Minor, optional: this sounds like a limitation. I wonder if `limitations.adoc` would not be a better place for mentioning these?

Reply:
Latest commit moves the wording to `limitations.adoc`.